airbase/pom.xml

```diff
@@ -781,6 +781,18 @@
                             <requirePropertyDiverges>
                                 <property>project.description</property>
                             </requirePropertyDiverges>
+                            <requireJavaVendor>
```

Comment:
This doesn’t belong in airbase. It’s a choice we made for Trino, but it may not be appropriate for all downstream projects.

Reply:
Are there projects like that?

Reply:
I've dropped this rule if it's questionable. PTAL at the rest of the changes